framework/core/src/Extension/Console/ToggleExtensionCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extension\Console;
+
+use Flarum\Console\AbstractCommand;
+use Flarum\Extension\ExtensionManager;
+
+class ToggleExtensionCommand extends AbstractCommand
+{
+    /**
+     * @var ExtensionManager
+     */
+    protected $extensionManager;
+
+    public function __construct(ExtensionManager $extensionManager)
+    {
+        $this->extensionManager = $extensionManager;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('extension:enable')
+            ->setAliases(['extension:disable'])
+            ->setDescription('Enable or disable an extension.')
+            ->addArgument('extension-id', null, 'The ID of the extension to enable or disable.');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function fire()
+    {
+        $name = $this->input->getArgument('extension-id');
+        $enabling = $this->input->getFirstArgument() === 'extension:enable';
+
+        if ($this->extensionManager->getExtension($name) === null) {
+            $this->error("There are no extensions by the ID of '$name'.");
+
+            return;
+        }
+
+        switch ($enabling) {
+            case true:
```

Comment:
I feel like an if/else would suffice here? I don't see how we're going to get any more cases than that.

Reply:
honestly true, I've been using the php8 `match` so often lately that my brain tries to produce something similar 😆

Reply:
Ah yes, pattern matching is indeed wonderful!

*suspiciously drops https://ocaml.org/docs/data-types on the way out*